rllib/core/rl_module/apis/inference_only_api.py

```diff
@@ -0,0 +1,56 @@
+import abc
+from typing import List
+
+
+class InferenceOnlyAPI(abc.ABC):
+    """An API to be implemented by RLModules that have an inference-only mode.
+
+    Only the `get_non_inference_attributes` method needs to get implemented for
+    a RLModule to have the following functionality:
+    - On EnvRunners (or when self.config.inference_only=True), RLlib will remove
```

Comment:
Can we indent these? It makes it easier to read.

Reply:
I'm not sure it'll pass the doctests, but let's try ...